### DISCUSSION TITLE
Wireframes examples

### DISCUSSION THREAD

**Discussion by aguilardias** [vuetifyjs/vuetify#12431] [Q&A] [answered]
I see that now in documentation, there are some wireframes examples, 
https://vuetifyjs.com/en/getting-started/wireframes/
where can i see the code of each example?
Where have gone the older examples, templates like youtube, google keep, login page, etc.. those examples helped a lot

**Comment by nekosaur** [comment, marked as answer]
You can click on the blue github icon in the lower right of each example to get the source code.

For the older examples, you can see them at https://v15.vuetifyjs.com/en/framework/pre-made-layouts. However keep in mind that they are for version 1.5 so they might need some changes to work properly in 2.x

**Comment by jacekkarczmarczyk** [comment]
You can find 2.x layouts in older commits: https://github.com/vuetifyjs/vuetify/tree/v2.3.0/packages/docs/src/layouts/layouts/demos
